frontend/src/components/RecentOrderCard/index.tsx

```diff
@@ -0,0 +1,167 @@
+import {Typography, styled, Box} from '@mui/material';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Chip from '@mui/material/Chip/Chip';
+import busIcon from '../../assets/bus-icon.svg';
+import arrowIcon from '../../assets/arrowIcon.svg';
+import scheduleIcon from '../../assets/schedule-icon.svg';
+
+export default function RecentOrderCard() {
```

Comment:
![image](https://github.com/bsoc-bitbyte/busify/assets/68912239/cece4186-a27c-4bff-8d24-4cbefd18f1a8)
Try to also optimize it for mobile view.